src/http-strategies/Express.js

```diff
@@ -190,9 +190,9 @@ function buildRequestObject(req, allowTunneling) {
   });
 }
 
-function hasBody(req) {
+function hasNonEmptyBody(req) {
   return req.headers["transfer-encoding"] !== undefined
-    || !isNaN(req.headers["content-length"]);
+    || (!isNaN(req.headers["content-length"]) && Number(req.headers["content-length"]) > 0);
```

Comment:
Don't think you really need the `Number()` call here, since invoking `>` will do that coercion anyway. Also, can you add a comment in `Request` that notes that hasBody means "has a (non-empty) body"?

Otherwise, this looks great! Thanks.